src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -828,7 +828,7 @@ for n,sFile in ipairs( tApis ) do
     end
 end
 
-if turtle then
+if turtle and fs.isDir( "rom/apis/turtle" ) then
```

Comment:
Well there were cheacks for `if pocket and fs.isDir( "rom/apis/pocket" ) then` and 'if commands and fs.isDir( "rom/apis/command" ) then'. I was under belief that this is accidentally committed.